extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css

```diff
@@ -65,25 +65,54 @@
   opacity: 0;
 }
 
+.i-amphtml-lbv-top-bar.overflow {
+  background: rgba(0,0,0,0.7);
+}
+
 .i-amphtml-lbv-desc-box {
   position: absolute !important;
   left: 0 !important;
   right: 0 !important;
   bottom: 0 !important;
-  max-height: 30vh !important;
-  padding: 10px;
-  overflow-x: hidden;
-  overflow-y: auto;
+  overflow-x: hidden !important;
   color: #ffffff;
-  background-color: rgba(0,0,0,0.7);
   opacity: 1;
   transition: opacity 1s;
 }
 
+.i-amphtml-lbv-desc-box.standard {
+  max-height: 100px;
```

Comment:
we should avoid `em` since it is relative to ancestor's font-size. Given lightbox-viewer is always child of body, it does not matter much in this case but `rem` is preferred.